kubernetes/samples/scripts/common/domain-template.yaml

```diff
@@ -98,3 +98,29 @@ spec:
   # The number of managed servers to start for unlisted clusters
   # replicas: 1
 
+  %MII_PREFIX%configuration:
+    # secrets: [ %DOMAIN_UID%-rcu-access ]
+
+  %MII_PREFIX%  model:
+      #
+      # Optional configmap for additional models and variable files
+      #
+  %MII_CONFIG_MAP_PREFIX%    configMap: %MII_CONFIG_MAP%
+
+      #
+      # wdt domain type for model in image case
+      # Valid WDT_DOMAIN_TYPE_VALUES are 'WLS', 'JRF', and 'RestrictedJRF'
+      #
+  %MII_PREFIX%    domainType: %WDT_DOMAIN_TYPE%
+  %MII_PREFIX%    runtimeEncryptionSecret: %WEBLOGIC_CREDENTIALS_SECRET_NAME%
+
+    #opss:
```

Comment:
Shall we use a prefix to control the OPSS part of the configuration? The prefix can be determined by the value of domainHomeSourceType and domainType, I assume.

Reply:
Just realized that this template is for the non-MII samples. I think I is probably clearer if we don't support the MII use case using these samples.

Reply:
@vanajamukkara Please follow up, and close this comment once the work is done. Thanks!

Reply:
Vanaj made a change for this.